README.md

```diff
@@ -44,6 +44,14 @@ $environment = Environment::createCommonMarkEnvironment();
 // Add this extension
 $environment->addExtension(new SmartPunctExtension());
 
+// Set your configuration
+$config = [
+    'smartpunct' => [
+        'double_quote_opener' => '„',
+        'double_quote_closer' => '“',
+    ],
+];
```

Comment:
Can you please make this show all four options and use the (current) default characters?